### DISCUSSION TITLE
<client-only>: Gone, Replaced, or Coming?

### DISCUSSION THREAD

**Discussion by midblue** [nuxt/nuxt#18314] [Questions] [answered]
Hello! I've noticed that in the Nuxt 3 beta, trying to use `<client-only>` to wrap elements now results in this warning:
```
[Vue warn]: Failed to resolve component: client-only
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
```

Is there any plan in the works for how to render components only on the client side? Thanks!

**Comment by danielroe** [comment, marked as answer]
You can follow https://github.com/nuxt/framework/issues/646.

**Reply by midblue** [reply]
Ah! Of course! I forgot to search issues, and only searched discussions. Sorry!
